index.test-d.ts

```diff
@@ -1,55 +1,139 @@
-import {expectType, expectError} from 'tsd';
-import Emittery = require('.');
+import { expectType, expectError } from "tsd";
+import Emittery = require(".");
 
 // emit
 {
 	const ee = new Emittery();
-	ee.emit('anEvent');
-	ee.emit('anEvent', 'some data');
+	ee.emit("anEvent");
```

Comment:
Don't do unrelated changes. Applies to a lot of places in this commit.

Reply:
I was pretty sure it was actually `xo` doing this which is why I included it, but maybe my editor's format on save option is broken. Should TypeScript be using singles or doubles?

Reply:
Single-quotes. It's not XO that's doing this.

Reply:
Ah, it is because xo is out of date, which I see you are trying to resolve. Let me know if I can help!